package.json

```diff
@@ -11,13 +11,14 @@
     "lint-fix": "eslint . --ext .js,.jsx,.ts,.tsx --fix",
     "clean": "lerna run clean && lerna exec yarn rimraf tsconfig.tsbuildinfo && lerna clean --yes && yarn rimraf node_modules",
     "build": "lerna run build",
-    "production-build": "yarn --frozen-lockfile && lerna run build --concurrency 3 --stream",
-    "dev-build": "yarn && lerna run build",
+    "production-build": "yarn --frozen-lockfile --ignore-scripts && lerna run build --concurrency 3 --stream",
```

Comment:
Would we need a "postinstall" npm script for production similar to what you've added for amplify-dev?

Reply:
good catch, will update that

Reply:
Actually this is okay as-is. The production-build is used in the pipeline and then the next step is publish to verdaccio and then the tests run by installing the CLI from verdaccio which will run the post-install script

Also, calling the post-install command can't be done as part of the production-build step because it doesn't actually put the amplify command on the path

Reply:
Okay, let’s make sure we have an E2E test covering the post-install script/command if there’s not one already.

Reply:
discussed offline. Added a better error message for this case, but existing tests cover it